tasks/cachet.yml

```diff
@@ -9,6 +9,11 @@
   delay: 10
   when:
     - cachet_deploy
+  environment:
```

Comment:
I would consider it as useless here

Reply:
PIP makes use of the proxy, as it needs it to access to Internet to get the pip packages.
So, I think is mandatory to have this here.